BGL/include/CGAL/boost/graph/named_params_helper.h

```diff
@@ -503,6 +503,50 @@ CGAL_DEF_GET_INITIALIZED_INDEX_MAP(face, typename boost::graph_traits<Graph>::fa
         > ::type  type;
     };
 
+    template<typename PointRange, typename NamedParameters>
+    class GetScanAngleMap
```

Comment:
The helper class is useless IMO as you can use `Constant_property_map` and copy/paste the `Lookup_named_param_def` directly in the code.